File: PRAK504-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int reverse (int a) {
    int b, n=0;
    while ( a != 0 ) {
        b = a % 10;
        n = b + n * 10;
        a /= 10; 
    } return n ; }

    int main () {
        int A, B;
        scanf ("%d %d", &A, &B);
        A = reverse (A);
        B = reverse (B);
        int C = A + B;
        printf ("%d\n", reverse (C));
    }

File: PRAK504-2210817120007-NursyahnaPuteri.py
def reverse (a) :
    n = 0
    while  int (a) != 0 : 
        b = int (a) % 10 
        n = n * b + 10
        a = int (a) / 10
    return n

A, B = map (int, input ().split ())
A = reverse (A)
B = reverse (B)
C = A + B
print (reverse (C))